frontend/src/app/modules/common/editable-toolbar-title/editable-toolbar-title.component.ts

```diff
@@ -206,4 +211,8 @@ export class EditableToolbarTitleComponent implements OnInit, OnChanges {
       el.classList.remove('-error');
     }
   }
+
+  private toggleToolbarButtonVisibility(hidden:boolean) {
+    jQuery('.toolbar-items').toggleClass('hidden-for-mobile', hidden);
```

Comment:
This is not an angular way of doing things although it works. 
An alternative would be to have a class on the editable toolbar title being toggled based on the focus. The css could then take care of the hiding.

Reply:
I agree that is no good style. Unfortunately the toolbar-items are not part of the editableToolbar, so they are not easily reachable via a class on the title. There are ways to achieve this more angular-wayish but I assume this would be ugly as well as we have to access the parent of both elements.